### PR TITLE
1.19.4 Support...?

### PR DESCRIPTION
**Pull request by ccorp2002**
# Pull Request

## Brief description.
I have implemented 1.19.4 Support, and I believe its working correctly. I have ran it on my test server, and all the features; biome colors, etc appear to function.

### What Issues Does This Fix?
Adds 1.19.4 Support, as much anticipated (6.3.0 Branch).

## Licensing

- [x] I am the original author of this code, and I am willing to release it
  under [GPLv3](https://www.gnu.org/licenses/gpl-3.0.en.html).
- [ ] I am not the original author of this code, but it is in public domain or
  released under [GPLv3](https://www.gnu.org/licenses/gpl-3.0.en.html) or a
  compatible license.
    

## Goal of the PR

1.19.4 Support!

## Affects of the PR

1.19.4 Support?

#### Types of changes

- [ ] Bug Fix 
- [ ] Build system 
- [ ] Documentation 
- [x] New Feature 
- [ ] Performance 
- [ ] Refactoring 
- [ ] Repository 
- [ ] Revert 
- [ ] Style 
- [ ] Tests 
- [ ] Translation 

#### Compatiblity

- [ ] Breaking
  change 
- [x] Non-Breaking change.
    

#### Contribution Guidelines.

- [x] I have read
  the [`CONTRIBUTING.md`](https://github.com/PolyhedralDev/Terra/blob/master/CONTRIBUTING.md)
  document in the root of the git repository.
- [x] My code follows the code style for this
  project. 

#### Documentation

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

#### Testing

- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: platforms/bukkit/nms/src/main/java/com/dfsek/terra/bukkit/nms/v1_19_R3/AwfulBukkitHacks.java.txt
Comment (duplexsystem) [inline platforms/bukkit/nms/src/main/java/com/dfsek/terra/bukkit/nms/v1_19_R3/AwfulBukkitHacks.java.txt:1]: This is an unnecessary file

**Review by duplexsystem — CHANGES_REQUESTED** [review]

File: platforms/bukkit/nms/src/main/java/com/dfsek/terra/bukkit/nms/v1_19_R3/AwfulBukkitHacks.java.txt
Comment (ccorp2002) [reply platforms/bukkit/nms/src/main/java/com/dfsek/terra/bukkit/nms/v1_19_R3/AwfulBukkitHacks.java.txt:1]: Removed.

**Review by dfsek — APPROVED** [review]
LGTM, going to test this and then I can do Fabric tonight. Then will merge and release.

**Comment by dfsek** [comment]
i fogor :skull: 

and also busy

**Comment by pontaoski** [comment]
prod? i'd like to contribute folia support but this being in is kinda a dependency (and submitting PRs dependent on other PRs is pain)

**Comment by dfsek** [comment]
Fabric loads into world on 1.19.4 now, both bukkit and fabric are crashing after chunk gen post-pack reload, though. looks like it has something to do with tags.

**Review by dfsek — APPROVED** [review]
Fixed fabric reloading, merging into 6.3.0 now.

**Pull request merged**